releasenotes/notes/enable-snmp-device-scan-command-d77dca95b30cb3e8.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Add a new snmp device scan command.
```

Comment:
```suggestion
    Add a new SNMP device scan command.
```

Reply:
updated in [da67a53](https://github.com/DataDog/datadog-agent/pull/29553/commits/da67a538811f8099128e8d53f35f8f7d745497f8)